docs/index.asciidoc

```diff
@@ -0,0 +1,31 @@
+:plugin: snmp
+:type: integration
+:no_codec:
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: %VERSION%
+:release_date: %RELEASE_DATE%
+:changelog_url: %CHANGELOG_URL%
+:include_path: ../../../../logstash/docs/include
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="plugins-{type}s-{plugin}"]
+
+=== SNMP Integration Plugin
+
+include::{include_path}/plugin_header.asciidoc[]
```

Comment:
The `plugin_header.asciidoc` file include can stay.  Technically, the integration plugin itself (but not its component plugins) behaves like a traditional (input, output, codec, filter) plugin.  Only the `plugin_header-integration.asciidoc` header file causes problems.